#!/usr/bin/env python
PACKAGE = "bender_arm_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("speed", double_t, 1, "Joint speed", 1.0, 0.01, 1.0)
gen.add("p_gain", int_t, 2, "P Gain", 10, 0, 254)
gen.add("i_gain", int_t, 4, "I Gain", 5, 0, 254)
gen.add("d_gain", int_t, 8, "D Gain", 0, 0, 254)
gen.add("torque_limit", double_t, 16, "Torque limit", 1.0, 0.01, 1.0)
gen.add("torque_enable", bool_t, 32, "Torque enable", True)

exit(gen.generate(PACKAGE, "bender_arm_control", "MotorMXParams"))